src/main/java/uk/gov/justice/services/test/DomainTest.java

```diff
@@ -100,4 +107,25 @@ public static JsonNode jsonNodeWithoutMetadataFrom(final JsonNode jsonEvent) {
     public static JsonNode generatedEventAsJsonNode(final Object generatedEvent) {
         return OBJECT_MAPPER.valueToTree(generatedEvent);
     }
+
+    public static List<JsonNode> generatedEventAsJsonNodeList(final List<Object> generatedEvent) {
+        final List<JsonNode> acutalEvents = new ArrayList<>();
+        for (Object event : generatedEvent) {
```

Comment:
This looks like a job for streams....:

return generatedEvent.stream()
    .map(OBJECT_MAPPER::valueToTree)
    .collect(toList());